docs/installation.rst

```diff
@@ -96,6 +96,14 @@ E-MAIL
 
   Defines maximum number of e-mails that are sent with command ``send_emails_batch``.
 
+.. attribute:: EMAIL_BATCH_SEND_ATTEMPTS_COUNT
+
+  Defines maximum number of attempts for sending one e-mail message. Default value is ``3``.
+
+.. attribute:: EMAIL_BATCH_MAX_SECONDS_TO_SEND
+
+  Defines maximum number of seconds for tring to send e-mail message that ended in a ``ERROR`` state. Default value is ``60 * 60`` (1 hour).
```

Comment:
Defines maximum number of seconds to try to send an e-mail message that ended in an ``ERROR`` state.